test/integration/targets/collections_runtime_pythonpath/aliases

```diff
@@ -0,0 +1,4 @@
+posix
+shippable/posix/group4
+skip/python2.6
+skip/python2.7
```

Comment:
@abadger ah, py2 doesn't fail because we skip it

Reply:
Why is Python 2.7 being skipped?

Reply:
No specific reason. I was playing w/ py3 only at first and wanted to make it work before going further.